server/community/queries.ts

```diff
@@ -116,3 +124,78 @@ export const updateCommunity = (inputValues, updatePermissions) => {
 		return filteredValues;
 	});
 };
+
+export const getCountOfPubsOrCollections = async (userId: string, communityId: string) => {
+	const promises = [
+		Member.count({
+			where: {
+				communityId,
+				userId,
+			},
+		}),
+
+		Member.count({
+			where: {
+				userId,
+			},
+			include: [
+				{
+					model: Pub,
+					as: 'pub',
+					where: {
+						communityId,
+					},
+				},
+			],
+		}),
+
+		Member.count({
+			where: {
+				userId,
+			},
+			include: [
+				{
+					model: Collection,
+					as: 'collection',
+					where: {
+						communityId,
+					},
+				},
+			],
+		}),
+
+		PubAttribution.count({
+			where: {
+				userId,
+			},
+			include: [
+				{
+					model: Pub,
+					as: 'pub',
+					where: {
+						communityId,
+					},
+				},
+			],
+		}),
+
+		CollectionAttribution.count({
+			where: {
+				userId,
+			},
+			include: [
+				{
+					model: Collection,
+					as: 'collection',
+					where: {
+						communityId,
+					},
+				},
+			],
+		}),
+	];
+	const counts = await Promise.all(promises);
+	// indicates if user is present
```

Comment:
Probably okay to (1) remove this comment and (2) return the `counts.some(...)` value directly without an intermediate variable.